docs/Meadow/Getting_Started/Hello_World/index.md

```diff
@@ -175,103 +151,95 @@ namespace HelloMeadow
             }
         }
 
-        void ShowColorPulse(Color color, int duration = 1000)
+        void ShowColorPulse(Color color, TimeSpan duration)
         {
-            onboardLed.StartPulse(color, (uint)(duration / 2));
-            Thread.Sleep(duration);
-            onboardLed.Stop();
-        }
-
-        void ShowColor(Color color, int duration = 1000)
-        {
-            Console.WriteLine($"Color: {color}");
-            onboardLed.SetColor(color);
+            onboardLed.StartPulse(color, (duration / 2));
             Thread.Sleep(duration);
             onboardLed.Stop();
         }
     }
 }
-
 ```
 
 ### Meadow Namespaces
 
-Let's break this down into pieces, first; the Meadow namespaces:
+Let's break down parts of this class into pieces, first; the Meadow namespaces:
```

Comment:
Small point - technically incorrect use of a semicolon. But if we're going to use one it should go after pieces

Reply:
Yeah, should have fixed that while I was editing. Good catch. There were a few others like that.

Reply:
Fixed.